NetKAN/SpearofLonginus.netkan

```diff
@@ -0,0 +1,19 @@
+{
+     "spec_version" : "v1.4",
+     "identifier"   : "SpearofLonginus",
+     "name"         : "Spear of Longinus",
+     "version"      : "0.8",

```

Comment:
This will give problems for future releases. as every future version will be 0.8